pkg/credentialprovider/azure/azure_credentials.go

```diff
@@ -92,37 +94,49 @@ func (a *acrProvider) Enabled() bool {
 		return false
 	}
 
-	servicePrincipalToken, err := azure.GetServicePrincipalToken(a.config, a.environment)
+	a.servicePrincipalToken, err = azure.GetServicePrincipalToken(a.config, a.environment)
 	if err != nil {
 		glog.Errorf("Failed to create service principal token: %v", err)
 		return false
 	}
 
+	glog.V(1).Infof("Creating a registry (control plane) client")
```

Comment:
That was left over aggressive, unneeded logging. I've simply removed the line now. All log line levels are 4 or greater.